src/locales/trending-topics-fallback.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
You might consider moving the fallback links out of json and into plain html in the trending topics hbs file, then toggling between the fallback div and the atomic div with css. 

Since we're only using the trending topics on the one page, it would be nice to pull it out of the js bundle. At the end of the day, it's a small amount of content though, so not a huge deal.

Reply:
You are totally right. I haven't worked with the UI bundle for so long, I forgot I can do that. Let me change them to HTML

Reply:
I moved these links into .hbs. Also made them into relative links so they don't work in the preview site, but should work in a regular site.

<img width="583" alt="Screenshot 2024-06-04 at 4 35 34 PM" src="https://github.com/mulesoft/docs-site-ui/assets/10934908/08274fdf-b126-443d-809d-55e5746835c0">